src/main/java/core/basesyntax/ColorSupplier.java

```diff
@@ -1,7 +1,12 @@
 package core.basesyntax;
 
+import java.util.Random;
+
 public class ColorSupplier {
-    public String getRandomColor() {
-        return null;
+    public static String getRandomColor() {
```

Comment:
Avoid using static methods as it's generally considered a bad practice. Instead, create an instance of the class to call its methods.

Reply:
not fixed - shouldn't be static